.github/workflows/scandir.yml

```diff
@@ -61,3 +61,19 @@ jobs:
           echo "::error:: Expected file $notexpect found in ${{ steps.two.outputs.files }}"
           exit 1
         fi
+
+    - name: Run ShellCheck
+      uses: ./
+      id: three
+      with:
+        scandir: './testfiles/scandir'
+        all_scripts: true
+
+    - name: Verify check
+      run: |
+        expect="testfiles/scandir/unknown-shebang"
+
+        if [[ ! "${{ steps.two.outputs.files }}" =~ $expect ]];then
+          echo "::error:: Expected file $expect not found in ${{ steps.two.outputs.files }}"
+          exit 1
+        fi
```

Comment:
This is a good suggestion, but I was following the style of whoever wrote these tests in the first place. I feel like a rewrite of this test for maintainability would lead to rewrite of other test cases, and that would quickly snowball out of the scope of this PR. I am open to rewriting the test syntax if the maintainer of this repository deems it a good idea.